### ISSUE TITLE
Android 6+ - ArgumentError: Error #3500

### ISSUE THREAD

**Issue by kryrnrj** [saumitrabhave/qr-zbar-ane#71]
Hi, This is the best QR ane of all time. It works like a charm on Android and IOS. But I have problem with android 6+. I create simple scanner instance. But it throws an error like this "ArgumentError: Error #3500: The extension context does not have a method with the name setConfig (or startPreview etc.)..... I tried on android 4.4 and it works so fast. Please help me. I want to use this ane in my project. Am i missing something or is there a problem with ane?

My code is simple like that:
`this._scanner=new Scanner;
this._scanner.setConfig(Symbology.QRCODE,Config.ENABLE,1);			
this._scanner.addEventListener(ScannerEvent.SCAN,onScan);	
this._scanner.startPreview(_currentCam,0,this._label.height,this._group.width,(this._group.height)-(this._label.height+this._button.height));`

**Comment by saumitrabhave** [comment]
Can you please share adb logcat output? for Android 6+ devices.

**Comment by kryrnrj** [comment]
Hi thanks for your reply.  I guess I found the problem. I am using AIR SDK 24 and my manifest file contains that line for permissions:<uses-sdk android:targetSdkVersion="23"/>. That's why it gives an error. I deleted this line but it still gives error. Finally i changed my air sdk to 23 and deleted the line. It works for now. 

adb logcat output:

`02-21 14:43:37.659 18235 18235 I QRExtension: createContext()
02-21 14:43:37.659 18235 18235 I QRExtensionContext: Reset Scanner
02-21 14:43:37.662 18235 18235 E linker  : /data/app/air.com.fernus.soruqrreader-1/lib/arm/libiconv.so: has text relocations
02-21 14:43:37.662 18235 18235 W System.err: java.lang.UnsatisfiedLinkError: dlopen failed: /data/app/air.com.fernus.soruqrreader-1/lib/arm/libiconv.so: has text relocations
02-21 14:43:37.663 18235 18235 W System.err:    at java.lang.Runtime.loadLibrary(Runtime.java:372)
02-21 14:43:37.663 18235 18235 W System.err:    at java.lang.System.loadLibrary(System.java:1076)
02-21 14:43:37.663 18235 18235 W System.err:    at net.sourceforge.zbar.ImageScanner.<clinit>(ImageScanner.java:35)
02-21 14:43:37.663 18235 18235 W System.err:    at com.sbhave.nativeExtension.QRExtensionContext.resetScanner(QRExtensionContext.java:116)
02-21 14:43:37.663 18235 18235 W System.err:    at com.sbhave.nativeExtension.QRExtension.createContext(QRExtension.java:39)
02-21 14:43:37.663 18235 18235 W System.err:    at com.adobe.air.customHandler.callTimeoutFunction(Native Method)
02-21 14:43:37.663 18235 18235 W System.err:    at com.adobe.air.customHandler.handleMessage(customHandler.java:22)
02-21 14:43:37.663 18235 18235 W System.err:    at android.os.Handler.dispatchMessage(Handler.java:102)
02-21 14:43:37.663 18235 18235 W System.err:    at android.os.Looper.loop(Looper.java:234)
02-21 14:43:37.663 18235 18235 W System.err:    at android.app.ActivityThread.main(ActivityThread.java:5526)
02-21 14:43:37.663 18235 18235 W System.err:    at java.lang.reflect.Method.invoke(Native Method)
02-21 14:43:37.663 18235 18235 W System.err:    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
02-21 14:43:37.663 18235 18235 W System.err:    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)`
